### PR TITLE
Add unit tests for Helm values generator

### PR DESCRIPTION
**Pull request by markgoddard**
Related: #13

### CODE REVIEW

File: internal/pkg/provider/helm/values_test.go [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+type ValueList = []interface{}
+
+func TestHelmValuesGenerator_GenerateValues_success(t *testing.T) {
+	tests := []struct {
+		name      string
+		trustZone *trust_zone_proto.TrustZone
+		want      Values
+	}{
+		{
+			name: "tz1 no apb or federation",
```
Comment (nialdaly) [inline internal/pkg/provider/helm/values_test.go:29]: Just a small question, is `abp` referring to attestation policy binding?

File: internal/pkg/provider/helm/values_test.go [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+			wantErrString: "an unknown trust provider profile was specified: invalid",
+		},
+		{
+			name: "unknown attestation policy",
+			trustZone: func() *trust_zone_proto.TrustZone {
+				tz := fixtures.TrustZone("tz1")
+				tz.AttestationPolicies[0].Policy = "invalid"
+				return tz
+			}(),
+			wantErrString: "failed to find attestation policy invalid in local config",
```
Comment (nialdaly) [inline internal/pkg/provider/helm/values_test.go:270]: More of a small nitpick but maybe we could rename `invalid` here to be `invalid-ap`? Just for verbosity 😃

File: internal/pkg/provider/helm/values_test.go [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+		},
+		{
+			name: "unknown federated trust zone",
+			trustZone: func() *trust_zone_proto.TrustZone {
+				tz := fixtures.TrustZone("tz1")
+				tz.Federations[0].To = "invalid"
```
Comment (nialdaly) [inline internal/pkg/provider/helm/values_test.go:276]: Similar to the comment above, perhaps `invalid-tz`?

File: internal/pkg/provider/helm/values_test.go (hunk 1 above)
Comment (markgoddard) [reply internal/pkg/provider/helm/values_test.go:29]: Yes. Is it clear enough or would you prefer something else? binding?

File: internal/pkg/provider/helm/values_test.go (hunk 2 above)
Comment (markgoddard) [reply internal/pkg/provider/helm/values_test.go:270]: :+1:

**Review by nialdaly — APPROVED** [review]
Looks good! 👌

**Pull request merged**